wikiclass/feature_lists/wikidatawiki.py

```diff
@@ -76,9 +76,13 @@ def simplify_target(target):
         return target
     elif hasattr(target, "toTimestr"):
         return target.toTimestr()
+    elif isinstance(target, dict) and 'text' in target:
```

Comment:
This is a fair point.  I should really say "WTF wikidata/pywikibase none of this is documented!"